README.md

```diff
@@ -24,15 +26,25 @@ In this challenge, create a web API around the following resources: `Projects` a
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
-- [ ] Mention two parts of Express that you learned about this week.
+- [x] Mention two parts of Express that you learned about this week.
+
+Two parts of express that I learned about are res.status(xxx) and req.body. I also learned that Express is middleware and can also be used to break the application into routers.
+
+- [x] Describe Middleware?
+
+Middleware is the software that connects network-based requests generated by a client to the back-end data the client is requesting.
```

Comment:
Also, middle ware is the functions you can insert into requests. Think of a multi step process, you can insert any middleware you'd like into the request flow.